app/models/manageiq/providers/lenovo/physical_infra_manager/parser/component_parser.rb

```diff
@@ -4,43 +4,88 @@ module ManageIQ::Providers::Lenovo
   # to a MiQ format
   #
   class PhysicalInfraManager::Parser::ComponentParser
-    #
-    # Returns a hash containing the structure described on dictionary
-    # and with the values in the source.
-    #
-    # @param source     - Object that will be parse to a hash
-    # @param dictionary - Hash containing the instructions to translate the object into a Hash
-    #
-    # @see ParserDictionaryConstants
```

Comment:
This is more a reminder to get rid of that reference when the `ParserDictionaryConstants` is deleted